libraries/helpers.rb

```diff
@@ -303,9 +303,11 @@ def platform_supports_dcv?
 end
 
 def aws_domain
-  # Set the aws domain name
+  # Get the aws domain name
   aws_domain = "amazonaws.com"
-  aws_domain = "#{aws_domain}.cn" if node['cluster']['region'].start_with?("cn-")
+  aws_domain = "amazonaws.com.cn" if node['cluster']['region'].start_with?("cn-")
```

Comment:
nitpick: it's cleaner to have a switch case

Reply:
I agree will fix in a follow up PR.